modin/backends/base/query_compiler.py

```diff
@@ -33,11 +32,10 @@
 
 
 def _get_axis(axis):
-    def axis_getter(self):
-        ErrorMessage.default_to_pandas(f"DataFrame.get_axis({axis})")
-        return self.to_pandas().axes[axis]
-
-    return axis_getter
+    if axis == 0:
+        return lambda self: self._modin_frame.index
+    else:
+        return lambda self: self._modin_frame.columns
```

Comment:
The `_get_axis`function with the same implementation is already placed in modin/backedns/pandas/query_compiler.py. Could you, please, exclude duplicating?